x-pack/plugins/cases/server/routes/api/cases/comments/patch_comment.ts

```diff
@@ -25,51 +25,66 @@ import {
   ENABLE_CASE_CONNECTOR,
 } from '../../../../../common/constants';
 import { CommentPatchRequestRt, throwErrors, User } from '../../../../../common/api';
-import { CaseServiceSetup } from '../../../../services';
+import { CaseService, AttachmentService } from '../../../../services';
 
 interface CombinedCaseParams {
-  service: CaseServiceSetup;
+  attachmentService: AttachmentService;
+  caseService: CaseService;
   client: SavedObjectsClientContract;
   caseID: string;
   logger: Logger;
   subCaseId?: string;
 }
 
 async function getCommentableCase({
-  service,
+  attachmentService,
+  caseService,
   client,
   caseID,
   subCaseId,
   logger,
 }: CombinedCaseParams) {
   if (subCaseId) {
     const [caseInfo, subCase] = await Promise.all([
-      service.getCase({
+      caseService.getCase({
         client,
         id: caseID,
       }),
-      service.getSubCase({
+      caseService.getSubCase({
         client,
         id: subCaseId,
       }),
     ]);
     return new CommentableCase({
+      attachmentService,
+      caseService,
       collection: caseInfo,
-      service,
       subCase,
       soClient: client,
       logger,
     });
   } else {
-    const caseInfo = await service.getCase({
+    const caseInfo = await caseService.getCase({
       client,
       id: caseID,
     });
-    return new CommentableCase({ collection: caseInfo, service, soClient: client, logger });
+    return new CommentableCase({
```

Comment:
I am wondering about this one, maybe `AttachToCase`

Reply:
Yeah I struggled with naming this class haha

Reply:
I am gonna leave it as it is and we can rethink it when we start working with sub cases again.